### PR TITLE
Support Transit Gateway(PER) in hypershift-hosted-powervs 

### PR DESCRIPTION
**Pull request by Neha-dot-Yadav**
Support for PER solution for powervs platform via transit gateway is present - [PR](https://github.com/openshift/hypershift/pull/3277)
This PR updates the workflow to leverage this for` hypershift-hosted-powervs `cluster creation and deletion.

### CODE REVIEW

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse release-openshift-origin-installer-launch-hypershift-hosted

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse release-openshift-origin-installer-launch-hypershift-powervs

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by aman4433** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-powervs-cb-test

File: ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml [hunk 1]
```diff
@@ -68,9 +68,12 @@ ref:
...
+  - name: POWERVS_TRANSIT_GATEWAY
     default: ""
-    documentation: "Cloud Connection in given zone.This flag is used to reuse an existing Cloud Connection resource for cluster's infra"
+    documentation: "Transit Gateway in IBM Cloud. This flag is used to reuse an existing Transit Gateway resource for cluster's infra"
+  - name: TRANSIT_GATEWAY_LOCATION
+    default: ""
+    documentation: "Transit Gateway Location to be used for the PowerVS setup."
   - name: CONTROL_PLANE_NODE_SELECTOR
     default: ""
     documentation: "Set nodeSelector for contorl plane pods. If the provided release image is not multi-arch, this is mandatory (e.g., `kubernetes.io/arch=amd64`)."
```
Comment (prb112) [inline ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml:79]:
```suggestion
    documentation: "Set nodeSelector for control plane pods. If the provided release image is not multi-arch, this is mandatory (e.g., `kubernetes.io/arch=amd64`)."
```

small spelling change

Comment (Neha-dot-Yadav) [reply ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml]: correct it, thanks!

**Review by bear-redhat — APPROVED** [review]

**Pull request merged**